website/docs/reference/resource-configs/pre-hook-post-hook.md

```diff
@@ -192,6 +192,10 @@ See: [Apache Spark docs on `ANALYZE TABLE`](https://spark.apache.org/docs/latest
 
 </File>
 
+import SQLCompilationError from '/snippets/_render-method.md';
+
+<SQLCompilationError />
```

Comment:
i'd move this above so it sits nicely after the 'why would i use hooks' section. maybe after the paragraph